### ISSUE TITLE
Voiceover ignores readonly and aria-readonly properties of input fields

### ISSUE THREAD

**Issue by abaevbog** [zotero/zotero#4787]
Voiceover does not announce inputs marked as readonly via `<input readonly="true"/`> or `<input aria-readonly="true"/>` and erroneously instructs the user to type to edit text. This is an issue within firefox as wel (last seen in firefox 131). It is specific to voiceover - NVDA and JAWS properly handle readonly fields.

Discussed more in https://github.com/zotero/zotero/pull/3997
